std/log/handlers.ts

```diff
@@ -85,33 +87,121 @@ export class ConsoleHandler extends BaseHandler {
 
 export abstract class WriterHandler extends BaseHandler {
   protected _writer!: Writer;
-  private _encoder = new TextEncoder();
-
+  #encoder = new TextEncoder();
+
+  /** @deprecated - `this._writer.write()` is an async call.  Under heavy load
+   * testing has shown a significant loss of log messages being written.  Also,
+   * if the process terminates while writing a log, that log message is also
+   * lost.  It is recommended that handlers override this method with a
+   * synchronous implementation (see FileHandler).
+   */
```

Comment:
Just remove this method completely, no need to mark it as deprecated

Reply:
As the `WriteHandler` class is abstract, I've removed the `@deprecated` comment and changed this log method to be abstract too.  This will generate compile errors for anyone currently subclassing `WriterHandler` with a custom implementation rather than if I removed it entirely, which would cause `log()` fall back to `BaseHandler` with a noop log method which could confuse or cause issues.